### PR TITLE
created useragent class

### PR DESCRIPTION
**Pull request by ieguiguren**
In response to:
 Create Agent Provider #7

### CODE REVIEW

File: project/http/requests/useragent/userAgent.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import random
+
+class UserAgent:
+
+    def __init__(self, web_proxy_list=[]):
```
Comment (pgaref) [inline project/http/requests/useragent/userAgent.py]: web_proxy_list is never used

File: project/http/requests/useragent/userAgent.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        self.agent_file = '../data/user_agents.txt'
```
Comment (pgaref) [inline project/http/requests/useragent/userAgent.py]: I would prefer the user_agent.txt file to be a parameter

File: project/http/requests/useragent/userAgent.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            path to text file of user agents, one per line
+        """
+        useragents = []
+        with open(useragentsfile, 'rb') as uaf:
+            for ua in uaf.readlines():
+                if ua:
+                    useragents.append(ua.strip()[1:-1-1])
+        return useragents
+
+    def random_user_agent(self):
```
Comment (pgaref) [inline project/http/requests/useragent/userAgent.py]: I would rename the method to something more descriptive like: get_random_user_agent

File: project/http/requests/useragent/userAgent.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import random
+
+class UserAgent:
```
Comment (pgaref) [inline project/http/requests/useragent/userAgent.py]: Just for clarify lets rename class to something like UserAgentManager or UserAgentHandler

**Comment by pgaref** [comment]
Thanks again! Look good in general, I just posted a few comments in the code - feel free to comment :)

**Comment by ieguiguren** [comment]
Agree. Added all the suggestions to the code.

File: project/http/requests/useragent/userAgent.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    def get_random_user_agent(self):
+        """
+        useragents : string array of different user agents
+        :param useragents:
+        :return random agent:
+        """
+        user_agent = random.choice(self.useragents)
+        return user_agent
+
+    def first_user_agent(self):
```
Comment (pgaref) [inline project/http/requests/useragent/userAgent.py]: similar to the above this should be: get_first_user_agent

File: project/http/requests/useragent/userAgent.py [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        return self.useragents[0]
+
+    def last_user_agent(self):
```
Comment (pgaref) [inline project/http/requests/useragent/userAgent.py]: similar to the above this should be: get_last_user_agent

File: project/http/requests/useragent/userAgent.py [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+        return self.useragents[-1]
+
+    def len_user_agent(self):
```
Comment (pgaref) [inline project/http/requests/useragent/userAgent.py]: similar to the above this should be: get_user_agents_len

**Comment by pgaref** [comment]
Looks good. Thanks!

**Pull request merged**